drivers/hmis/app/graphql/mutations/assign_staff.rb

```diff
@@ -0,0 +1,45 @@
+#  Copyright 2016 - 2024 Green River Data Analysis, LLC
+#
+#  License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+#
+
+module Mutations
+  class AssignStaff < CleanBaseMutation
+    argument :household_id, ID, required: true
+    argument :assignment_type_id, ID, required: true
+    argument :user_id, ID, required: true
+
+    field :staff_assignment, Types::HmisSchema::StaffAssignment, null: true
+
+    def resolve(household_id:, assignment_type_id:, user_id:)
+      household = Hmis::Hud::Household.preload(:project).find_by(household_id: household_id, data_source_id: current_user.hmis_data_source_id)
```

Comment:
Since we only load a single record, I think the preload isn't needed. Also probably should use the viewable_by scope so we never even load a record the user can't see
```suggestion
      household = Hmis::Hud::Household.viewable_by(current_user).find_by(household_id: household_id, data_source_id: current_user.hmis_data_source_id)
```